src/libNode/FinalBlockProcessing.cpp

```diff
@@ -1329,22 +1209,17 @@ void Node::CommitForwardedTransactionBuffer()
                             .GetBlockNum())
         {
             it = m_forwardedTxnBuffer.erase(it);
+            continue;
         }
-        else if (it->first
-                 == m_mediator.m_txBlockChain.GetLastBlock()
-                        .GetHeader()
-                        .GetBlockNum())
+        else
         {
             for (const auto& entry : it->second)
             {
                 ProcessForwardTransactionCore(entry);
             }
             m_forwardedTxnBuffer.erase(it);
-            break;
-        }
-        else
-        {
-            it++;
+            continue;
         }
+        it++;
```

Comment:
This line seems never be executed, since the loop always be continued before here.

Reply:
Noted